modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala

```diff
@@ -34,15 +34,14 @@ import scala.math.abs
 import scala.collection.JavaConverters._
 
 trait EpicsCommand[F[_]] {
-  def post: F[ApplyCommandResult]
+  def post(timeout: Time): F[ApplyCommandResult]
```

Comment:
This is very good, a great improvement. Should we use `FiniteDuration` rather than `Time`?

Reply:
Why can't you let me be lazy ?